edge_computing/image-base-install/ibi-factory-image-based-install.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Performing an image-based installation for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-factory-image-based-install
+
+toc::[]
+
+Use the `openshift-install` installation program to create a live installation ISO for preinstalling {sno} on bare-metal hosts. For more information about downloading the installation program, see _Installation process_ in the _Additional resources_ section.
+
+The installation program takes a seed image URL, and other inputs such as a URL for an instance of {op-system-first}, and creates a live installation ISO that references these installation artifacts. You can then restart the host using the live installation ISO to begin the preinstallation of the host. When the preinstallation is complete, the host is ready to ship to a remote site for the final site-specific configuration and deployment.
```

Comment:
> such as a URL for an instance of {op-system-first}

I think we should remove this part, as this is probably a leftover from the ib-cli documentation. There is no such config option for the `openshift-install`, but there are other ones like the `seedVersion` or the `installationDisk`.

Reply:
Removed outdated reference - thanks